### PR TITLE
[10.0][MIG] hr_timesheet_task_required migration

### PR DESCRIPTION
**Pull request by benwillig**
This PR depends on #93.

### CODE REVIEW

**Comment by ThomasBinsfeld** [comment]
#93 is merged, need rebase

**Comment by benwillig** [comment]
@ThomasBinsfeld Done!

File: hr_timesheet_task_required/README.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========================
+Hr Timesheet Task Required
+==========================
+
+This module allow to set task on timesheet as a mandatory field.
```
Comment (ThomasBinsfeld) [inline hr_timesheet_task_required/README.rst]: @benwillig s/allow/allows

File: hr_timesheet_task_required/README.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/117/8.0
```
Comment (ThomasBinsfeld) [inline hr_timesheet_task_required/README.rst]: @benwillig s/8.0/10.0

File: hr_timesheet_task_required/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Hr Timesheet Task Required',
+    'summary': """
+        Set task on timesheet as a mandatory field""",
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV, Odoo Community Association (OCA)',
+    'website': 'https://www.acsone.eu/',
```
Comment (ThomasBinsfeld) [inline hr_timesheet_task_required/__manifest__.py]: @benwillig `'website': 'https://github.com/OCA/hr-timesheet',`

File: hr_timesheet_task_required/README.rst [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Adrien Peiffer <adrien.peiffer@acsone.eu>
+* Laurent Mignon <laurent.mignon@acsone.eu>
```
Comment (ThomasBinsfeld) [inline hr_timesheet_task_required/README.rst]: @benwillig Add yourself in contributors

**Review by ThomasBinsfeld — COMMENTED** [review]
Nitpicking comments, lgtm

**Comment by benwillig** [comment]
All done!

**Review by ThomasBinsfeld — APPROVED** [review]
Code and functional reviews :+1:

File: hr_timesheet_task_required/static/src/js/hr_timesheet.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+            if (id === false) {
+                self.dfm.set({display_invalid_fields: true});
+                return;
+            }
+            return self._super();
+        }
+
+    });
+
+});
```
Comment (LoisRForgeFlow) [inline hr_timesheet_task_required/static/src/js/hr_timesheet.js]: Maybe you should add the end lines in all files.

**Review by LoisRForgeFlow — APPROVED** [review]
Code review LGTM

**Review by adrienpeiffer — APPROVED** [review]
:+1: Code review and test

File: hr_timesheet_task_required/static/src/js/hr_timesheet.js (hunk 5 above)
Comment (benwillig) [reply hr_timesheet_task_required/static/src/js/hr_timesheet.js]: Done!

**Comment by benwillig** [comment]
This PR can be merged I think.

**Pull request merged**